queries/date.py

```diff
@@ -836,6 +867,22 @@ def tomorrowdate_answ(q: Query, result: Result) -> None:
     q.set_answer(response, answer, voice)
 
 
+def yesterday_date_answ(q: Query, result: Result) -> None:
+    """ Generate answer to a question of the form "Hvaða dagur var í gær?" etc. """
+    now = datetime.utcnow() - timedelta(days=1)
```

Comment:
Strictly speaking we should have a utility function that returns the current time in the user's current locale/location, and call that instead of `datetime.utcnow()` in the appropriate places.

Reply:
Yes. This is true. Been meaning to standardise stuff like this for a while. IMO the qmodule code needs some serious rethinking and refactoring, including stuff like this. Things are not as clear, straightforward, predictable and standardised as they should be. Let's sit down and improve the design of control flow, best practices etc. This is now about 10 thousand lines of code [!] and becoming unwieldy.